loaders/android/bootstrap.java.in

```diff
@@ -220,13 +243,13 @@ public class @SYS_APPNAME@ extends Activity implements @ANDROID_JAVA_IMPLEMENTS@
     }
   }
 
-  native void nativeInstanceInit();
+  native void nativeInstanceInit(String packageCodePath, String filesDir);
 }
 
 class  xGLSurfaceView extends GLSurfaceView {
   public xGLSurfaceView(Context context) {
     super(context);
-    setFocusable(true); 
+    setFocusable(true);
```

Comment:
All changes from here on out in this file are cosmetic (space at end of line). I guess the vimrc from #135 was never included anywhere?